src/main/java/by/exadel/internship/controller/SchedulingController.java

```diff
@@ -0,0 +1,48 @@
+package by.exadel.internship.controller;
+
+
+import by.exadel.internship.annotation.AdminAccessControl;
+import by.exadel.internship.dto.time_for_call.UserTimeSlotWithUserIdDTO;
+import by.exadel.internship.dto.UserDTO;
+import by.exadel.internship.service.SchedulingService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/scheduling")
+@Api(tags = "Endpoints for Scheduling")
+public class SchedulingController {
+
+    private final SchedulingService schedulingService;
+
+    @AdminAccessControl
+    @GetMapping("/interview-time")
+    @ApiOperation("Get all users with their free time for Form")
+    public List<UserDTO> getPossibleInterviewTime(@RequestParam("formId") UUID formId) {
+        return schedulingService.getAdminTimeForForm(formId);
+    }
+
+    @AdminAccessControl
+    @PostMapping("/save-interview-time")
+    @ApiOperation("Save interview time for Form")
+    public void saveInterviewForForm(@RequestParam(name = "formId") UUID formId,
```

Comment:
You have so good method name :) which prompts you where this endpoint should be. Move both endpoints to form controller. Possible path: POST/PUT `/{form}/interview`. And also if you start using `InterviewDTO` instead of `UserTimeSlotWithUserIdDTO` it can really simplify your huge validation on service layer. 

cc: @SergeyKoval  @viktarkolbik

Reply:
Fixed. I use UserTimeSlot because we need to delete this time if busy

Reply:
Dsno